spec/mspec/lib/mspec/guards/platform.rb

```diff
@@ -62,6 +62,12 @@ def self.wsl?
     WORD_SIZE
   end
 
+  C_LONG_SIZE = if defined?(RbConfig::SIZEOF[])
```

Comment:
I wonder what `WORD_SIZE` actually means given there is already `POINTER_SIZE`.
Maybe `WORD_SIZE` is meant to be `sizeof(long)` already but with a bad name?
Do you think it would make sense to basically rename :wordsize to :c_long_size (or make it an alias of it and still update all usages).

Reply:
Nice catch! In fact, it looks like the only usage of `wordsize?` left is `platform_spec`. Will try for a rename.